lib/auth/index.js

```diff
@@ -23,6 +23,15 @@ module.exports = settings => {
   const keycloak = new Keycloak({ store: settings.store }, config);
   const permissions = can(settings.permissions);
 
+  router.use((req, res, next) => {
+    if (!req.session && !settings.bearerOnly) {
+      const err = new Error('No session found');
+      err.status = 500;
```

Comment:
There's no need to manually set a status of 500. It's the default. I'd just `next(new Error('No session'))`

Reply:
I have no idea what you're talking about 🤷‍♀️